src/main/java/de/tum/in/www1/artemis/service/user/UserService.java

```diff
@@ -285,7 +285,7 @@ public User registerUser(UserDTO userDTO, String password) {
                 vcsUserManagementService.deactivateUser(savedNonActivatedUser.getLogin());
             }
             catch (VersionControlException e) {
-                log.error("An error occurred while registering GitLab user " + savedNonActivatedUser.getLogin() + ":", e);
+                log.error("An error occurred while registering GitLab user {}:", savedNonActivatedUser.getLogin(), e);
```

Comment:
```suggestion
                log.error("An error occurred while registering GitLab user {}: ", savedNonActivatedUser.getLogin(), e);
```

Reply:
I don't see a need for this, in fact I think that message should not end with a space.

Reply:
I wasn't really sure for this one, as some log.error ended with a space some didn't